### PR TITLE
Depend on ignition-utils1 cli component

### LINKED ISSUE #54: Add standalone executable for ign-plugin-info command
We updated the ign-transport `ign topic` and `ign service` commands to use standalone executables in https://github.com/ignitionrobotics/ign-transport/pull/216. We should do the same for the `ign plugin --info` command defined in the loader component:

* https://github.com/ignitionrobotics/ign-plugin/blob/ign-plugin1/loader/src/cmd/cmdplugin.rb.in

similar to https://github.com/ignitionrobotics/ign-transport/issues/247

### PR DESCRIPTION
**Pull request by scpeters**
# 🎉 New feature

Precursor to https://github.com/ignitionrobotics/ign-plugin/issues/54

## Summary

This adds a dependency on the `ignition-utils1::cli` component that will be used to resolve #54. This also changes any includes of `ignition/utilities/SuppressWarning.hh` to `ignition/utils/SuppressWarning.hh` and changes the exported target dependency accordingly. This may impact downstream packages if they are using the `ignition/utilities` headers implicitly from this package. I'll monitor the nightly builds.

## Test it

It should still compile like normal and pass all tests.

## Checklist
- [X] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [X] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [X] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [X] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

**Comment by chapulina** [comment]
> This may impact downstream packages if they are using the ignition/utilities headers implicitly from this package. I'll monitor the nightly builds.

I don't think any downstream packages are using `ign-plugin2`, so this may not manifest just yet

File: tutorials/02_installation.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ sudo apt install libignition-plugin-dev
 
 1. Install Ignition dependencies
   ```
-  sudo apt-get install libignition-cmake2-dev
+  sudo apt-get install libignition-cmake2-dev libignition-utils1-cli-dev
```
Comment (adlarkin) [inline tutorials/02_installation.md:46]: The `packages.apt` file for this repo also has `libignition-tools-dev`, should that be added here? https://github.com/ignitionrobotics/ign-plugin/blob/7f3df50589c996b88aba9a64b7e15f7f7097e40b/.github/ci/packages.apt#L2

**Comment by scpeters** [comment]
updating ign-plugin windows builds to use colcon in https://github.com/ignition-tooling/release-tools/pull/495

**Review by ahcorde — APPROVED** [review]

**Review by ahcorde — COMMENTED** [review]
When this PR https://github.com/ignition-tooling/release-tools/pull/495 is in. I will run the CI again

File: tutorials/02_installation.md (hunk 1 above)
Comment (scpeters) [reply tutorials/02_installation.md]: 17c2ce2

**Pull request merged**